grammar/BUILD

```diff
@@ -41,6 +43,51 @@ java_library(
     tags = ["maven_coordinates=com.vaticle.typeql:typeql-grammar:{pom_version}", "checkstyle_ignore"],
 )
 
+rust_grammar_adapter(
+    name = "rust-grammar",
+    input = "TypeQL.g4",
+    output = "TypeQLRust.g4",
+)
+
+antlr(
+    name = "rust-src",
+    srcs = [":rust-grammar"],
+    language = "Rust",
+    visitor = True,
+    package = "typeqlgrammar",
+)
+
+
+rust_library(
+    name = "typeql-grammar",
+    srcs = [":rust-src"],
+    deps = [
+        "@vaticle_dependencies//library/crates:antlr_rust",
+    ]
+)
+
+assemble_crate(
+    name = "assemble-crate",
+    target = ":typeql-grammar",
+    description = "TypeQL Grammar for Rust",
+    license = "AGPLv3",
+    deps = {
+        "antlr-rust": "0.2.0",
```

Comment:
When https://github.com/vaticle/bazel-distribution/issues/323 is implemented, we won't have to specify `deps` manually. It will be collected automatically from the `deps` field of the `rust_library` and `rust_binary` targets.